src/main/java/edu/wpi/grip/core/SocketHints.java

```diff
@@ -0,0 +1,105 @@
+package edu.wpi.grip.core;
+
+
+import org.bytedeco.javacpp.opencv_core.Size;
+import org.bytedeco.javacpp.opencv_core.Mat;
+import org.bytedeco.javacpp.opencv_core.Point;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Supplier;
+
+
+/**
+ * Create default socket hints
+ */
+public final class SocketHints {
```

Comment:
Something so that [this](https://github.com/WPIRoboticsProjects/GRIP/pull/177/files#diff-35295a6e1bb5368122de3caef3564decR34) could be replaced with `SocketHints.createEnumSocketHint("Type", Type.class)`.